kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/internal/expressions/DefaultExpressionEvaluatorSuite.scala

```diff
@@ -582,6 +582,69 @@ class DefaultExpressionEvaluatorSuite extends AnyFunSuite with ExpressionSuiteBa
       "LIKE expression has invalid escape sequence"))
   }
 
+  test("evaluate expression: starts with") {
+    val col1 = stringVector(Seq[String]("one", "two", "t%hree", "four", null, null, "%"))
+    val col2 = stringVector(Seq[String]("o", "t", "T", "4", "f", null, null))
+    val schema = new StructType()
+      .add("col1", StringType.STRING)
+      .add("col2", StringType.STRING)
+    val input = new DefaultColumnarBatch(col1.getSize, schema, Array(col1, col2))
+
+    val startsWithExpressionLiteral = startsWith(new Column("col1"), Literal.ofString("t%"))
+    val expOutputVectorLiteral =
```

Comment:
can you check in spark the behavior of `starts_with(null, 't%')` is null as well?

Reply:
Ran 
SELECT startswith(NULL,'t%'); -> NULL 
SELECT startswith('t%',NULL); -> NULL

also found the below documentation "If expr or startExpr is NULL, the result is NULL."
https://learn.microsoft.com/en-us/azure/databricks/sql/language-manual/functions/startswith#examples